src/lib/components/FilterForm.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useCallback, useEffect } from 'react';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import _ from 'lodash';
+import { useDispatch, useSelector } from 'react-redux';
+import { Grid, Typography } from '@material-ui/core';
+import type { RootStateType } from '../../store/rootReducer';
+import { ExpertRegionType } from '../types';
+import { setExpertsRegionsFilter } from '../../modules/experts/store/expertsSlice';
+
+export type CheckedType = {
+  [key: string]: boolean;
+};
+
+export interface IInitLocalState {
+  [key: string]: boolean;
+}
+
+export interface ICheckboxes {
+  [key: number]: boolean;
+}
```

Comment:
```suggestion
export interface ICheckboxes {
  [key: string]: boolean;
}
```
you could use just one interface here, with string keys to match initial state

Reply:
fixed